cli/js/lib.deno.ns.d.ts

```diff
@@ -1368,6 +1374,30 @@ declare namespace Deno {
    */
   export function shutdown(rid: number, how: ShutdownMode): void;
 
+  /** UNSTABLE: new API
+   * A socket is a generic transport listener for message-oriented protocols
+   */
+  export interface Socket extends AsyncIterator<[Uint8Array, Addr]> {
```

Comment:
Please follow Go's naming conventions and call this `UDPConn`

https://golang.org/pkg/net/#UDPConn